src/module-vsbridge-indexer-catalog/Model/Indexer/Action/Product.php

```diff
@@ -57,6 +65,17 @@ public function rebuild($storeId = 1, array $productIds = [])
 
                 unset($product['required_options']);
                 unset($product['has_options']);
+
+                $productObject = new \Magento\Framework\DataObject();
+                $productObject->setData($product);
+
+                $this->eventManager->dispatch(
+                    'elasticsearch_product_build_entity_data_after',
+                    ['data_object' => $productObject]
+                );
+
+                $product = $productObject->getData();
+
```

Comment:
I am not sure why we need to add an event here? What is the purpose for that?
If you like to add more fields/data to product type, you can create new data provider.

Reply:
@afirlejczyk Hi,  
Thanks for the return I did not see that feature!  
Maybe it can more explicit in the documentation on how to extend the extension 🙂 
I will close this pull request.   
Thank you!